tests/cases/fourslash/refactorConvertToEs6Module_import_nameFromModuleSpecifier.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+
+// @Filename: /a.js
+////const [] = /*a0*/require/*b0*/("a-b");
+////const [] = /*a1*/require/*b1*/("0a");
+////const [] = /*a2*/require/*b2*/("1a");
+
+goTo.select("a0", "b0");
+edit.applyRefactor({
+    refactorName: "Convert to ES6 module",
+    actionName: "Convert to ES6 module",
+    actionDescription: "Convert to ES6 module",
+    newContent: `import aB from "a-b";
+const [] = aB;
+import A from "0a";
+const [] = A;
+import _A from "1a";
```

Comment:
ideally we would have all the import statements at the top and not interleaved with the statements

Reply:
Similar comment in https://github.com/Microsoft/TypeScript/pull/19916#discussion_r150630648
I don't think that would be easy to do as we are replacing all of the `... = require(...)` statements in place -- we're not guaranteed that these will all be in a row at the top of the file anyway.
This case should be pretty rare though since it's unusual to destructure a module into an array.